src/Resources/config/services.yaml

```diff
@@ -0,0 +1,42 @@
+parameters:
+    landingi_toggle_redis_connection:
+        schema: 'tcp'
+        host: '127.0.0.1'
+        port: 6379
```

Comment:
Jest opcja w tej konfiguracji, zeby ustawić client'a envem? 

W adminie jest np:
```
dsn: "%env(REDIS_URL)%"
```

Reply:
Tak, w readme jest przykład właśnie z envem. Symfony to automatycznie rozwiąże. Tutaj to jest tylko default.